roles/openshift_examples/defaults/main.yml

```diff
@@ -6,7 +6,7 @@ openshift_examples_load_db_templates: true
 openshift_examples_load_xpaas: "{{ openshift_deployment_type in ['enterprise','openshift-enterprise','atomic-enterprise','online']  }}"
 openshift_examples_load_quickstarts: true
 
-examples_base: /usr/share/openshift/examples
+examples_base: "{% if is_containerized %}{{ openshift.common.config_base }}{% else %}/usr/share/openshift{% endif %}/examples"
```

Comment:
If I am not misunderstanding `is_containerized` vs `is_atomic`, this should be `is_atomic`. We are only prohibited from writing there on atomic hosts.